android/src/test/java/com/instabug/flutter/SurveysApiTest.java

```diff
@@ -0,0 +1,157 @@
+package com.instabug.flutter;
+
+import static com.instabug.flutter.util.MockResult.makeResult;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.mockStatic;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+
+import com.instabug.flutter.generated.SurveysPigeon;
+import com.instabug.flutter.modules.SurveysApi;
+import com.instabug.flutter.util.GlobalMocks;
+import com.instabug.library.Feature;
+import com.instabug.survey.Survey;
+import com.instabug.survey.Surveys;
+import com.instabug.survey.callbacks.OnDismissCallback;
+import com.instabug.survey.callbacks.OnShowCallback;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.MockedStatic;
+
+import java.sql.Array;
+import java.util.ArrayList;
+import java.util.List;
+
+import io.flutter.plugin.common.BinaryMessenger;
+
+
+public class SurveysApiTest {
+    private final SurveysApi mApi = new SurveysApi(null);
```

Comment:
**suggestion:** Instead of initializing this with `null`, we can pass the right dependencies, to make sure nothing misbehaves in the future.
```suggestion
    private final BinaryMessenger mMessenger = mock(BinaryMessenger.class);
    private final SurveysPigeon.SurveysFlutterApi flutterApi = new SurveysPigeon.SurveysFlutterApi(mMessenger);
    private final SurveysApi api = new SurveysApi(flutterApi);
```